From 69f797c22d0d10963efa67638f2ee0be5eae2053 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Tue, 18 Mar 2014 13:14:22 -0400
Subject: [PATCH] mtools: Disable iconv use on target

Since the /usr/lib/gconv directory isn't copied over to the target
with some toolchains, iconv_open will fail. This removes character set
conversion in the target version of mtools.

Signed-off-by: Frank Hunleth <fhunleth@troodon-software.com>
---
 package/mtools/mtools.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/package/mtools/mtools.mk b/package/mtools/mtools.mk
index 268c3003d7..5ffa305f33 100644
--- a/package/mtools/mtools.mk
+++ b/package/mtools/mtools.mk
@@ -26,6 +26,11 @@ HOST_MTOOLS_CONF_ENV = \
 ifeq ($(BR2_PACKAGE_LIBICONV),y)
 MTOOLS_DEPENDENCIES += libiconv
 MTOOLS_CONF_ENV += LIBS=-liconv
+# We have no host dependencies
+HOST_MTOOLS_DEPENDENCIES =
+else
+# external toolchain doesn't install gconv. This fixes a runtime error.
+MTOOLS_CONF_OPTS += ac_cv_header_iconv_h='no'
 endif
 
 # Package does not build in parallel due to improper make rules
-- 
2.25.1

